llvm/lib/Target/WebAssembly/Disassembler/WebAssemblyDisassembler.cpp

```diff
@@ -238,7 +239,7 @@ MCDisassembler::DecodeStatus WebAssemblyDisassembler::getInstruction(
         auto *WasmSym = cast<MCSymbolWasm>(Sym);
         WasmSym->setType(wasm::WASM_SYMBOL_TYPE_FUNCTION);
         const MCExpr *Expr = MCSymbolRefExpr::create(
-            WasmSym, MCSymbolRefExpr::VK_WASM_TYPEINDEX, getContext());
+            WasmSym, WebAssemblyMCExpr::VK_TYPEINDEX, getContext());
```

Comment:
What is the VariantKind in these resulting expressions?  i.e. what geKind() now return instead of the old `VK_WASM_..` values?